### DISCUSSION TITLE
Export: option to fail on any cell error

### DISCUSSION THREAD

**Discussion by fonsp** [JuliaPluto/PlutoSliderServer.jl#49] [Ideas]
Originally suggested on zulip by @kir0ul:

When using PlutoSliderServer in CI to generate HTML exports, e.g. using https://github.com/JuliaPluto/static-export-template , it would be useful to have an option to make the CI fail if a notebook errored.

**Comment by j-fu** [comment]
In this context  the ability to test/render a notebook with the development version instead of  a release would be useful, so I link 
https://github.com/fonsp/Pluto.jl/issues/1788 here.
